kiosk/src/Components/ScanQR.tsx

```diff
@@ -26,11 +26,16 @@ const ScanQR: React.FC<IProps> = ({ kiosk }) => {
     const checkUrl = async () => {
         const input = document.getElementById("kiosk-share-link") as HTMLInputElement;
         const inputValue = input.value;
```

Comment:
I believe this would include incidental whitespace, so might be worth a ?.trim();
```suggestion
        const inputValue = input.value?.trim();
```